content/en/blog/news/2022-gsod.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Project ideas for Google Season of Docs 2022 ☀️"
+date: 2022-03-23
+draft: false
+description: >
+  Project ideas for GSoD 2022!
+categories: [blog]
+keywords: [Community, 2022]
+slug: "GSoD2022-ideas"
+aliases: []
+author: Jenkins X maintainers
+---
+
+We have put together some project ideas as part of our application to participate in the Google Season of Docs 2022 program.
+
+### Project ideas
+
+#### 1. Better documentation for real world use cases
+
+##### Problem
+
+- End users of Jenkins X are unable to find information on how to use Jenkins X for real world use cases which includes:
+  - Production best practices for
+  - Scanning images in Jenkins X pipelines
+  - Tests (Integration, e2e)
+  - Artifact management
+  - Multi cluster
+  - Integration with other tools in the cloud native sector
+
+##### How would we measure success?
+
+- Decrease in the number of github issues raised by end users on documentation related to common real world use cases
+- Short 1-2 minute videos on common tasks that end users of Jenkins X would like to perform.
+
+##### Recommended skills
+
+- Must have: Experience with github
+- Nice to have: Basic experience with docker compose (local development scripts use compose) and video editing skills.
+
+##### Mentors/Volunteers
+
+- [Ankit D Mohapatra](https://github.com/ankitm123)
+- [Christopher Vig](https://github.com/babadofar)
+- [Mårten Svantesson](https://github.com/msvticket)
+- [Tom Hobson](https://github.com/tomhobson)
+
+#### 2. Reorganize documentation to make it more user friendly and easy to search
```

Comment:
Split in two - 
- reorganize documentation
- improve searchability
While these tasks are related, they can also be done separately  
 Search would benefit from reorganization, but we may consider adding extra metadata etc. to improve it even more.

Reply:
The reason I grouped them as one, is because we may go with only one doc student, and I think each contributor can pick one project.

Reply:
Fine!